ext/pgsql/tests/pg_set_chunked_rows_mode.phpt

```diff
@@ -0,0 +1,49 @@
+--TEST--
+PostgreSQL pg_set_chunked_rows_mode
+--EXTENSIONS--
+pgsql
+--SKIPIF--
+<?php 
+include("inc/skipif.inc");
+if (!function_exists("pg_set_chunked_rows_mode")) die("pg_set_chunked_rows_mode unsupported");
```

Comment:
The tests it borked otherwise due to an invalid SKIPIF section
```suggestion
if (!function_exists("pg_set_chunked_rows_mode")) die("skip pg_set_chunked_rows_mode unsupported");
```